### PR TITLE
[CLI-2710] Update packaging during release

### PR DESCRIPTION
**Pull request by brianstrauch**
Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok

What
----
Introduce a new target to update the packaging repo with the current CLI version after every release.

Test & Review
-------------
Tested with `DRY_RUN=true`.

### CODE REVIEW

File: mk-files/release.mk
Comment (sgagniere) [inline mk-files/release.mk:202]: It looks like one of these should have a `v` and one should not. This code seems to be setting both to the same value, unless I'm reading it incorrectly.

Comment (brianstrauch) [reply mk-files/release.mk:202]: > It looks like one of these should have a v and one should not.

Correct.

**Review by sgagniere — APPROVED** [review]

**Comment by brianstrauch** [comment]
Waiting until next release to merge, since we don't want to open a PR in packaging this week since we missed the CP 7.5 deadline.

**Pull request merged**
